site2/docs/io-kafka-source.md

```diff
@@ -17,19 +17,47 @@ The configuration of the Kafka source connector has the following properties.
 | `fetchMinBytes` | long|false | 1 | The minimum byte expected for each fetch response. |
 | `autoCommitEnabled` | boolean |false | true | If set to true, the consumer's offset is periodically committed in the background.<br/><br/> This committed offset is used when the process fails as the position from which a new consumer begins. |
 | `autoCommitIntervalMs` | long|false | 5000 | The frequency in milliseconds that the consumer offsets are auto-committed to Kafka if `autoCommitEnabled` is set to true. |
 | `heartbeatIntervalMs` | long| false | 3000 | The interval between heartbeats to the consumer when using Kafka's group management facilities. <br/><br/>**Note: `heartbeatIntervalMs` must be smaller than `sessionTimeoutMs`**.|
 | `sessionTimeoutMs` | long|false | 30000 | The timeout used to detect consumer failures when using Kafka's group management facility. |
 | `topic` | String|true | " " (empty string)| The Kafka topic which sends messages to Pulsar. |
-|  `consumerConfigProperties` | Map| false | " " (empty string) | The consumer configuration properties to be passed to consumers. <br/><br/>**Note: other properties specified in the connector configuration file take precedence over this configuration**. |
+| `consumerConfigProperties` | Map| false | " " (empty string) | The consumer configuration properties to be passed to consumers. <br/><br/>**Note: other properties specified in the connector configuration file take precedence over this configuration**. |
 | `keyDeserializationClass` | String|false | org.apache.kafka.common.serialization.StringDeserializer | The deserializer class for Kafka consumers to deserialize keys.<br/> The deserializer is set by a specific implementation of [`KafkaAbstractSource`](https://github.com/apache/pulsar/blob/master/pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaAbstractSource.java).
 | `valueDeserializationClass` | String|false | org.apache.kafka.common.serialization.ByteArrayDeserializer | The deserializer class for Kafka consumers to deserialize values.
 | `autoOffsetReset` | String | false | "earliest" | The default offset reset policy. |
 
+### Schema Management
+
+This Kafka Source connector applies the Schema to the topic depennding on the data type that is present on the Kafka topic.
+We are detecting the data type from the `keyDeserializationClass` and `valueDeserializationClass` configuration parameters.
+
+For instance if the `valueDeserializationClass` is `org.apache.kafka.common.serialization.StringDeserializer` then we set Schema.STRING() as schema type
+on the Pulsar topic.
+
+If `valueDeserializationClass` is `io.confluent.kafka.serializers.KafkaAvroDeserializer` we download the AVRO schema from the Confluent Schema Registry®
+and we set it properly on the Pulsar topic. In this case you have to set `schema.registry.url` inside of the `consumerConfigProperties` configuration entry
+of the source.
```

Comment:
Avoid the first-person pronouns `I` and `we`. Use the second-person pronoun `you` as much as possible.

Reply:
got it